caravel/assets/javascripts/global_functions.js

```diff
@@ -0,0 +1,28 @@
+var $ = window.$ = require('jquery');
```

Comment:
I can move showApiMessage() and toggleCheckbox() into utils.js. 
But we still need to include some .entry.js in list.html, right? Should I still include global_functions.entry.js (or maybe rename it to list.entry.js) and call util.toggleChekbox() in this file? Or do you have any other idea?